AV-45749: Update sample queries (#35)
couchbaselabs/docs-devex: modules/guides/pages/prep-statements.adoc

.Query
[source,sqlpp]
----

> include::example$prepared/elixir-prep-statements.n1ql[tag=twonamed]

Reviewer: These example files will be migrated to docs-devex as part of DOC-10865. To avoid broken includes in the meantime, specify the server component.

```suggestion
include::server:settings:example$param-names.n1ql[tag=statement]
```

.Query
[source,sqlpp]
----

> include::example$prepared/elixir-prep-statements.n1ql[tag=twonumbered]

Reviewer: These example files will be migrated to docs-devex as part of [DOC-10865](https://issues.couchbase.com/browse/DOC-10865). To avoid broken includes in the meantime, specify the server component.

```suggestion
include::server:settings:example$param-numbers.n1ql[tag=statement]
```

.Query
[source,sqlpp]
----

> include::example$prepared/elixir-prep-statements.n1ql[tag=twounnumbered]

Reviewer: These example files will be migrated to docs-devex as part of [DOC-10865](https://issues.couchbase.com/browse/DOC-10865). To avoid broken includes in the meantime, specify the server component.

```suggestion
include::server:settings:example$param-positions.n1ql[tag=statement]
```

--
To supply values for placeholder parameters:

//* If you are using the cbq shell, use the `\SET` command to set the parameters, then run the statement.

* If you are using the {sqlpp} REST API, specify the parameters in the request body or the query URI, alongside the statement.

> * If you are using the Query Workbench, click btn:[options] to display the Query Options window, set the parameters, then run the statement.

Reviewer 2: ```suggestion
* If you are using the Query Workbench, click btn:[options] to display the Query Options dialog, set the parameters, then run the statement.
```

> * If you are using the Query Workbench, use the btn:[options] button to display the Query Options window, set the parameters, then run the statement.

Reviewer 2: ```suggestion
* If you are using the Query Workbench, click btn:[options] to display the Query Options window, set the parameters, then run the statement.
```
